ecommerce/models.py

```diff
@@ -111,6 +111,61 @@ def type_string(self):
         """
         return self.content_type.model
 
+    @property
+    def title(self):
+        """
+        Helper property to return a string representation of the product title,
```

Comment:
nit: please remove `Helper property to`

Reply:
this is how this file was written, I just followed the convention. For reference see property `type_string `

Reply:
not sure, either it is a convention or not. above of `type_string` a `latest_version` does not have this convention. anyway it is just a suggestion, we can ignore that.